debian/control

```diff
@@ -14,7 +14,7 @@ Description: securedrop client for qubes workstation
 
 Package: securedrop-export
 Architecture: all
-Depends: ${misc:Depends}, python3, udisks2, cups, printer-driver-brlaser, printer-driver-hpcups, system-config-printer, xpp, libcups2, gnome-disk-utility, libreoffice,
+Depends: ${misc:Depends}, python3, udisks2, cups,  avahi-daemon, systemd, system-config-printer, xpp, libcups2, gnome-disk-utility, libreoffice,
```

Comment:
The systemd dependency is only because we're installing a file into its path, correct?

I think we need a dependency on `ipp-usb`, right?